src/resonanceReconstruction/rmatrix.hpp

```diff
@@ -52,14 +54,13 @@ namespace rmatrix {
   #include "resonanceReconstruction/rmatrix/ParticleChannel.hpp"
   #include "resonanceReconstruction/rmatrix/ParticleChannelData.hpp"
 
-  // resolved resonance information
```

Comment:
Is "resolved" considered to be redundant, so that someone who speaks of "resonances" implicitly means "resolved resonances" unless otherwise specified? I'm just asking because I'm not really familiar with the terminology here.

Reply:
Eventually, "resolved" will become redundant once all "unresolved" resonances have been removed. It's related to the energy resolution of a measurement and whther or not we can distinguish individual resonances. When the individual resonances can be distinguished, they are said to be resolved. They are unresolved if they cannot be distinguished. There's more to it than that, but it would take to long to explain, we should keep that for a time when we talk on the phone.